solarwinds_apm/api/__init__.py

```diff
@@ -113,19 +115,20 @@ def solarwinds_ready(
      if not solarwinds_ready(wait_milliseconds=10000, integer_response=True):
         Logger.info("SolarWinds not ready after 10 seconds, no metrics will be sent")
     """
-    rc = Context.isReady(wait_milliseconds)
-    if not isinstance(rc, int) or rc not in OboeReadyCode.code_values():
-        logger.warning("Unrecognized return code: %s", rc)
-        return (
-            OboeReadyCode.OBOE_SERVER_RESPONSE_UNKNOWN[0]
-            if integer_response
-            else False
-        )
-    if rc != OboeReadyCode.OBOE_SERVER_RESPONSE_OK[0]:
-        logger.warning(OboeReadyCode.code_values()[rc])
-
-    return (
-        rc
-        if integer_response
-        else rc == OboeReadyCode.OBOE_SERVER_RESPONSE_OK[0]
-    )
+    return True
```

Comment:
Nit: Could we add a TODO?

```suggestion
    # TODO: Reimplement in NH-104961
    return True
```